docs/generate_db_schema

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I recommend the fully explicit ```/usr/bin/python3``` here since that's what the Fedora packaging guidelines require.

Reply:
Even with `#!/usr/bin/env python3` the test is failing, because sqlalchemy_schemadisplay is only python 2. The only option how this could run is to use `#!/usr/bin/env python` or `#!/usr/bin/python`